PacketDotNet/Interfaces/ILogInactive.cs

```diff
@@ -1,9 +1,8 @@
-using System;
 #if DEBUG
 using log4net.Core;
 #endif
 
-namespace PacketDotNet
+namespace PacketDotNet.Interfaces
```

Comment:
Why place these interfaces into their own namespace? Is this .net recommended practices? I'd like to understand more about the motivation here.